### PR TITLE
add get action

### PR DESCRIPTION
**Pull request by wyangsun**

### CODE REVIEW

**Comment by LindsayHill** [comment]
Can you please increment the version in pack.yaml, and add an entry to CHANGES.md?

File: actions/get.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from lib.base import BaseRedisAction
+
+__all__ = [
+    'DelRedisAction'
+]
+
+
+class DelRedisAction(BaseRedisAction):
```
Comment (LindsayHill) [inline actions/get.py]: This class name should change - probably something like `GetRedisAction`

**Review by LindsayHill — CHANGES_REQUESTED** [review]
Needs `pack.yaml` version bump, and entry to CHANGES.md.

Also needs linting errors fixed - should just be a newline at the end of the file, and remove the whitespace at the end of line 11

File: actions/get.py (hunk 1 above)
Comment (wyangsun) [reply actions/get.py]: Sorry for my mistake.

**Comment by wyangsun** [comment]
updated, please help to review.

**Review by LindsayHill — APPROVED** [review]

**Comment by LindsayHill** [comment]
I think this is all OK now, but CircleCI is having an [outage](
https://status.circleci.com/incidents/yj497xcsqxxf). I'll wait for that to get fixed, then re-run a CircleCI check, then we can merge it.

**Pull request merged**